Yoast/wordpress-seo
# Remove prominent word indexable columns migration from the Internal Linking feature branch

## Summary


This PR can be summarized in the following changelog entry:

* Removes migration for the prominent_word_version and prominent_word_vector_length in the feature/internal-linking-phrase-2 branch

## Test instructions

This PR can be tested by following these steps:

### Replication
* Remove the columns `prominent_word_version` and `prominent_word_vector_length` from the `wp-yoast-indexable` table of the database of your local WP site.
* Remove the migration `20190813111111` from the list in `wp_yoast_migrations`.
* Check out the `feature/internal-linking-phase-2` branch.
* Build the plugin.
* Open any page in your local environment (e.g., Post Editor).
* Verify that the columns `prominent_word_version` and `prominent_word_vector_length` appeared again in the `wp-yoast-indexable` table of the database.
### Testing
* Remove the columns `prominent_word_version` and `prominent_word_vector_length` from the `wp-yoast-indexable` table of the database of your local WP site.
* Remove the migration `20190813111111` from the list in `wp_yoast_migrations`.
* Check out the `13701-remove-prominent-word-indexable-columns-migration-from-internal-linking-feature-branch` branch.
* Build the plugin.
* Open any page in your local environment.
* Verify that the columns `prominent_word_version` and `prominent_word_vector_length` **did not** appear in the `wp-yoast-indexable` table of the database.

## UI changes

## Documentation

## Quality assurance


Fixes #13701
